modules/custom/az_news/az_news_feeds/az_news_feeds.routing.yml

```
@@ -0,0 +1,9 @@
+az_news_feeds.admin_importers:
+  path: '/admin/content/az-news-feeds'
+  defaults:
+    _form: '\Drupal\az_news_feeds\Form\AzNewsFeedsAdminForm'
+    link_id: 'az_news_feeds.admin_importers'
+    _title: 'Administer Importers'
```

Nitpick (a less generic title would be helpful here)
```suggestion
    _title: 'Configure UArizona News Importer'
```